jet_tracking/devices.py

```diff
@@ -746,6 +746,28 @@ class Parameters(Device, _TableMixin):
                          doc='Nozzle counter')
     nozzle_reprate = Cpt(EpicsSignal, ':NOZZLE_RepRate',
                          doc='Nozzle repetition rate')
+    mean = Cpt(EpicsSignal, ':ROI_mean',
+               doc='mean of calibration ROI image with jet')
```

Comment:
TODO: create constants directory/file with strings so we only have to change the value in one place in case it's used in multiple files/locations.

Reply:
Which strings? The PV prefixes and docstrings for Cpt's should only be used here.